src/main/java/yapper/app/MainWindow.java

```diff
@@ -0,0 +1,63 @@
+package yapper.app;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+/**
+ * Controller for the main GUI.
+ */
+public class MainWindow extends AnchorPane {
+    @FXML
+    private ScrollPane scrollPane;
+    @FXML
+    private VBox dialogContainer;
+    @FXML
+    private TextField userInput;
+    @FXML
+    private Button sendButton;
+
+    private Yapper yapper;
+
+    private Image userImage = new Image(this.getClass().getResourceAsStream("/images/you.png"));
+    private Image dukeImage = new Image(this.getClass().getResourceAsStream("/images/icon.png"));
```

Comment:
Did you forget to change this to yapperImage?

Reply:
Yes, woops. Will change.